library/maven

```diff
@@ -83,27 +78,37 @@ Directory: adoptopenjdk-11
 
 Tags: 3.6.3-adoptopenjdk-11-openj9, 3.6-adoptopenjdk-11-openj9, 3-adoptopenjdk-11-openj9
 Architectures: amd64, arm64v8, ppc64le, s390x
-GitCommit: 26ba49149787c85b9c51222b47c00879b2a0afde
+GitCommit: b243010db790a1ab8f6a43d26ae86a76cba86b55
 Directory: adoptopenjdk-11-openj9
 
-Tags: 3.6.3-adoptopenjdk-14, 3.6.3-adoptopenjdk, 3.6-adoptopenjdk-14, 3.6-adoptopenjdk, 3-adoptopenjdk-14, 3-adoptopenjdk, adoptopenjdk
+Tags: 3.6.3-adoptopenjdk-14, 3.6-adoptopenjdk-14, 3-adoptopenjdk-14
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
 GitCommit: 5768b638a84739d0ad85dbcbe42018075550b109
 Directory: adoptopenjdk-14
 
 Tags: 3.6.3-adoptopenjdk-14-openj9, 3.6-adoptopenjdk-14-openj9, 3-adoptopenjdk-14-openj9
 Architectures: amd64, ppc64le, s390x
-GitCommit: 5768b638a84739d0ad85dbcbe42018075550b109
+GitCommit: b243010db790a1ab8f6a43d26ae86a76cba86b55
 Directory: adoptopenjdk-14-openj9
 
+Tags: 3.6.3-adoptopenjdk-15, 3.6.3-adoptopenjdk, 3.6-adoptopenjdk-15, 3.6-adoptopenjdk, 3-adoptopenjdk-15, 3-adoptopenjdk, adoptopenjdk
+Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
+GitCommit: ecb160edac810c845116ef70ffbda8bfeca21075
+Directory: adoptopenjdk-15
+
+Tags: 3.6.3-adoptopenjdk-15-openj9, 3.6-adoptopenjdk-15-openj9, 3-adoptopenjdk-15-openj9
+Architectures: amd64, ppc64le, s390x
+GitCommit: b243010db790a1ab8f6a43d26ae86a76cba86b55
+Directory: adoptopenjdk-15-openj9
+
 Tags: 3.6.3-adoptopenjdk-8, 3.6-adoptopenjdk-8, 3-adoptopenjdk-8
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
```

Comment:
The failure is related to this version,`maven:3.6.3-adoptopenjdk-8` being `FROM adoptopenjdk:8-jdk-hotspot`, which no longer has an arm32v7 image (https://github.com/docker-library/official-images/pull/8953).

Reply:
I just needed to pull master 🤦